scoring/score.py

```diff
@@ -1,9 +1,57 @@
+DATA = {
```

Comment:
This is not a very precise variable name

Reply:
f0a2936